net/nexttrace/Makefile

```
@@ -19,6 +19,7 @@ PKG_BUILD_FLAGS:=no-mips16
 
 GO_PKG:=github.com/nxtrace/NTrace-core
 GO_PKG_LDFLAGS_X:=$(GO_PKG)/config.Version=v$(PKG_VERSION)
+GO_PKG_LDFLAGS:-checklinkname=0
```

```suggestion
GO_PKG_LDFLAGS:=-checklinkname=0
```

And move it above `GO_PKG_LDFLAGS_X`

---

> And move it above `GO_PKG_LDFLAGS_X`

Done~
thanks for the hints!